components/Slider/tests/snapshot/MDCSliderSnapshotTests.m

```diff
@@ -763,4 +763,26 @@ - (void)testDiscreteSliderWithTrackHeight10Points {
   [self generateSnapshotAndVerifyForView:self.slider];
 }
 
+- (void)testDiscreteSliderWithThumbAndValueLabel {
+  // When
+  [self makeSliderDiscrete:self.slider];
+  self.slider.value =
+      self.slider.minimumValue + (self.slider.maximumValue - self.slider.minimumValue) / 2;
+  self.slider.shouldDisplayDiscreteValueLabel = YES;
+  self.slider.shouldDisplayThumbWithDiscreteValueLabel = YES;
+  // In Thumbtrack's code, there is a check for verifying that the thumbtrack's width is larger
+  // than 1 point, otherwise it won't go into the main frame adjusting logic. This is to make sure
+  // that the scale transform of the slider's view isn't at its default of 0.001. Therefore this
+  // transform adjustment was made so it can let the logic know we are actually interacting with
+  // the thumb in the test.
+  UIView *valueLabel = [self.slider.thumbTrack valueForKey:@"_valueLabel"];
+  valueLabel.transform = CGAffineTransformIdentity;
```

Comment:
It's a bit odd that we need to do this in the snapshot test; is there a way for us to simulate the effect of touching the slider without modifying the transform?

Reply:
removed the transform